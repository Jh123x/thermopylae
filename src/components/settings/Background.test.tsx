import { test, expect, describe } from '@jest/globals';
import { render } from '@testing-library/react';
import BackgroundSettings from './Background';

describe('BackgroundSettings', () => {
    test('should render correctly', () => {
        const result = render(<BackgroundSettings />);
        expect(result.baseElement).toMatchSnapshot();
    });
});