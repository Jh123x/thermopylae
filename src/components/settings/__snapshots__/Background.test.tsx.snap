// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundSettings should render correctly 1`] = `
<body>
  <div>
    <div
      class="background-settings"
    >
      <div
        class="input-group"
      >
        <label>
          Background Image
        </label>
        <input
          type="file"
          value=""
        />
      </div>
      <div
        class="input-group"
      >
        <label>
          Particles
        </label>
        <p>
          <input
            checked=""
            type="checkbox"
          />
          <span>
            Enabled
          </span>
        </p>
      </div>
    </div>
  </div>
</body>
`;
